js/bitz.js

```diff
@@ -426,7 +426,7 @@ module.exports = class bitz extends Exchange {
         //          source:   "api"                             }
         //
         const ticker = this.parseTicker (response['data'], market);
-        let timestamp = this.parseMicrotime (this.safeString (response, 'microtime'));
+        let timestamp = this.safeInteger (response, 'time') * 1000;
```

Comment:
This loses the precision, right? Can we keep the microtime though? I mean, what was the bug, exactly?

Reply:
Yeah, precision is not as good as a working basis of microtime instead. 

`this.parseMicrotime (this.safeString (response, 'microtime'))`
Returned 1 additional number, ie timestamp was "15485909900" instead of "1547935268"

Reply:
And datetime was year 2422 or so

Reply:
Ill make a more proper fix in a couple minutes, found the fault

Reply:
Fixed total field in microtime function. Shows properly now;
`["timestamp"]=>
    int(1548593161059)
    ["datetime"]=>
    string(24) "2019-01-27T12:46:01.059Z"`